### ISSUE TITLE
base64 pre-config option

### ISSUE THREAD

**Issue by AstraLuma** [python-attrs/cattrs#552]
* cattrs version: 23.2.3
* Python version: 3.11
* Operating System: Linux

### Description

`cattrs.preconf.json.make_converter()` and `cattrs.preconf.ujson.make_converter()` use Base85.

While this _is_ more efficient, it's bad for interop in the sense that base64 is much more common. (eg, both Browsers and Node have it built-in.) (Yes, Base85 code can be found for basically any language, but you can often assume that Base64 will be standard library.)

### What I Did

```
try:
    from cattrs.preconf.ujson import make_converter
except ImportError:
    from cattrs.preconf.json import make_converter

converter = make_converter()
```

**Comment by Tinche** [comment]
Yeah I agree, if I could go back in time I'd have it be base64 by default. A configuration option for these converters sounds good.
